spec/services/api/v1/users_collection_spec.rb

```diff
@@ -23,8 +23,8 @@
         expect(collection.execute_query.size).to eq(3)
       end
 
-      it 'returns 5 columns' do
-        expect(collection.execute_query.first.size).to eq(5)
+      it 'returns 6 columns' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.